components/rf_bridge.rst

```diff
@@ -0,0 +1,198 @@
+RF Bridge Component
+===================
+
+.. seo::
+    :description: Instructions for setting up the RF Bridge in ESPHome.
+    :image: rf_bridge.jpg
+    :keywords: RF Bridge
+
+The ``RF Bridge`` Component provides the ability to send and receive 433MHz remote codes without hardware
+hacking the circuit board to bypass the ``efm8bb1`` MCU. This component implements the communcation protocol
+that the original ``efm8bb1`` firmware implements. The device is connected via the
+:doc:`UART bus </components/uart>`. The uart bus must be configured at the same speed of the module
+which is 19200bps.
+
+.. warning::
+
+    If you are using the :doc:`logger` make sure you disable the uart logging with the
+    ``baud_rate: 0`` option.
+
+.. figure:: images/rf_bridge-full.jpg
+    :align: center
+    :width: 60.0%
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    uart:
+      baud_rate: 19200
+
+    rf_bridge:
+      on_code_received:
+        - homeassistant.event:
+            event: esphome.rf_code_received
+            data:
+              sync: !lambda 'char buffer [10];return itoa(data.sync,buffer,16);'
+              low: !lambda 'char buffer [10];return itoa(data.low,buffer,16);'
+              high: !lambda 'char buffer [10];return itoa(data.high,buffer,16);'
+              code: !lambda 'char buffer [10];return itoa(data.code,buffer,16);'
+
+Configuration variables:
+------------------------
+
+- **uart_id** (*Optional*, :ref:`config-id`): Manually specify the ID of the UART hub.
+- **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
+- **on_code_received** (*Optional*, :ref:`Automation <automation>`): An action to be
+  performed when a code is received. See :ref:`rf_bridge-on_code_received`.
+
+.. _rf_bridge-on_code_received:
+
+``on_code_received`` Trigger
+----------------------------
+
+With this configuration option you can write complex automations whenever a code is
+received. To use the code, use a :ref:`lambda <config-lambda>` template, the code
+and the corresponding protocol timings are available inside that lambda under the
+variables named ``code``, ``sync``, ``high`` and ``low``.
+
+.. code-block:: yaml
+
+    on_code_received:
+      - homeassistant.event:
+          event: esphome.rf_code_received
+          data:
+            sync: !lambda 'char buffer [10];return itoa(data.sync,buffer,16);'
+            low: !lambda 'char buffer [10];return itoa(data.low,buffer,16);'
+            high: !lambda 'char buffer [10];return itoa(data.high,buffer,16);'
+            code: !lambda 'char buffer [10];return itoa(data.code,buffer,16);'
+
+
+.. rf_bridge-send_code_action:
```

Comment:
```suggestion
.. _rf_bridge-send_code_action:
```

Otherwise this won't be a valid ref link.

Reply:
I tried it locally and it was failing still. Seems like it still not working on CI